token/js/package.json

```diff
@@ -1,74 +1,86 @@
-        "deploy": "yarn docs && gh-pages --dist docs --dest token/js --dotfiles",
+        "nuke": "shx rm -rf node_modules package-lock.json || true",
+        "reinstall": "npm run nuke && npm install",
+        "clean": "shx rm -rf lib **/*.tsbuildinfo || true",
+        "build": "tsc --build --verbose tsconfig.all.json",
+        "watch": "tsc --build --verbose --watch tsconfig.all.json",
+        "release": "npm run clean && npm run build",
+        "fmt": "prettier --write '{*,**/*}.{ts,tsx,js,jsx,json}'",
+        "lint": "prettier --check '{*,**/*}.{ts,tsx,js,jsx,json}' && eslint --max-warnings 0 .",
+        "lint:fix": "npm run fmt && eslint --fix .",
         "example": "node --experimental-specifier-resolution=node --loader ts-node/esm examples/createMintAndTransferTokens.ts",
-        "test": "yarn test:unit && yarn test:e2e-built && yarn test:e2e-native && yarn test:e2e-2022",
+        "test": "npm run test:unit && npm run test:e2e-built && npm run test:e2e-native && npm run test:e2e-2022",
         "test:unit": "mocha test/unit",
         "test:e2e-built": "start-server-and-test 'solana-test-validator --bpf-program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA ../../target/deploy/spl_token.so --reset --quiet' http://localhost:8899/health 'mocha test/e2e'",
         "test:e2e-2022": "TEST_PROGRAM_ID=TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb start-server-and-test 'solana-test-validator --bpf-program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL ../../target/deploy/spl_associated_token_account.so --bpf-program TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb ../../target/deploy/spl_token_2022.so --reset --quiet' http://localhost:8899/health 'mocha test/e2e*'",
         "test:e2e-native": "start-server-and-test 'solana-test-validator --reset --quiet' http://localhost:8899/health 'mocha test/e2e'",
         "test:build-programs": "cargo build-sbf --manifest-path ../program/Cargo.toml && cargo build-sbf --manifest-path ../program-2022/Cargo.toml && cargo build-sbf --manifest-path ../../associated-token-account/program/Cargo.toml",
-        "docs": "shx rm -rf docs && NODE_OPTIONS=--max_old_space_size=4096 typedoc && shx cp .nojekyll docs/",
-        "fmt": "prettier --write '{*,**/*}.{js,ts,jsx,tsx,json}'",
-        "lint": "eslint --max-warnings 0 --ext .ts . && prettier --check '{*,**/*}.{js,ts,jsx,tsx,json}'",
-        "lint:fix": "eslint --fix --ext .ts . && yarn fmt",
-        "nuke": "shx rm -rf node_modules yarn.lock"
+        "deploy": "npm run deploy:docs",
+        "docs": "shx rm -rf docs && typedoc && shx cp .nojekyll docs/",
+        "deploy:docs": "npm run docs && gh-pages --dist token/js --dotfiles"
+    },
+    "peerDependencies": {
+        "@solana/web3.js": "^1.20.0"
     },
     "dependencies": {
         "@solana/buffer-layout": "^4.0.0",
         "@solana/buffer-layout-utils": "^0.2.0",
-        "@solana/web3.js": "^1.41.0"
+        "@solana/spl-memo": "^0.2.0",
```

Comment:
Shouldn't this be a dev dependency? Also, may as well specify 0.3.0 for the next version, right?

Reply:
This is an unbundled runtime dependency, so not a dev dependency. But I did change the version to 0.2.1 to match the next one.

Reply:
How is it a runtime dependency? It's only ever used in a test

Reply:
Oh, I was mistaken. I thought it was used by the new `MemoTransferInstruction` but it isn't. Fixed!